### PR TITLE
make netlify init work with netlify.toml, see #210 #267

### LINKED ISSUE #210: skip asking for build/deploy directory if netlify.toml exists, during `netlify init`
**- Do you want to request a *feature* or report a *bug*?**

minor feature

**- What is the current behavior?**

I have a netlify.toml in my root. when i run `netlify init` it still proceeds to ask me these useless questions

image

**- What is the expected behavior?**

Just dont ask

**- Please mention your node.js, and operating system version.**

not relevant

### PR DESCRIPTION
**Pull request by sw-yx**
**- Summary**

closes #210 and #267

**- Test plan**

https://netlify.slack.com/archives/CEV26F3FF/p1552942688000100

**- Description for the changelog**

when doing `netlify init`:

- if netlify.toml doesn't exist: offer to create one with the given settings
- if netlify.toml exists: use settings from netlify.toml as defaults

**- A picture of a cute animal (not mandatory but encouraged)**

https://twitter.com/HumansOfLate/status/1107013958207238145

### CODE REVIEW

File: src/utils/init/netlify-toml-template.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const makeNetlifyTOMLtemplate = ({ command, publish }) => `# example netlify.toml
+[build]
+  command = "${command}"
+  functions = "lambda" #  netlify-lambda reads this
```
Comment (DavidWells) [inline src/utils/init/netlify-toml-template.js]: I'm torn between `functions` vs `lambda` as the default here. https://github.com/search?o=desc&p=6&q=functions+filename%3Anetlify.toml&s=indexed&type=Code

Comment (sw-yx) [reply src/utils/init/netlify-toml-template.js]: ok i'll switch it

**Comment by sw-yx** [comment]
cc @bcomnes idk if u wanna review but i think this is ready

**Comment by bcomnes** [comment]
Hey sorry I'll review tomorrow AM.

**Comment by sw-yx** [comment]
oh of course wasnt meant to be urgent

**Comment by sw-yx** [comment]
lets dewit

**Pull request merged**